Client/src/renderer/views/command-centre/CommandCentre.vue

```diff
@@ -53,7 +53,21 @@ onUnmounted(async () => {
     UserServerAction.offRoomUpdate(updateRoom);
 });
 
-const sendCommand = async (): Promise<void> => {
```

Comment:
is it possible to define these functions as:

```ts
async function sendCommand(): Promise<void> {
   ...
}
```

That's slightly more readable IMO - unless there's a specific reason to keep them as consts